rs/crypto/internal/crypto_lib/basic_sig/ed25519/src/api/tests.rs

```diff
@@ -322,128 +316,6 @@ mod verify {
         }
     }
 
-    #[test]
-    fn should_correctly_verify_batches_of_signatures_using_different_keys_on_same_message(
```

Comment:
This test seems redundant with the test `batch_verification_works` in `ic-crypto-ed25519` with the exception that this test covers larger batches. (The current test in `ic-crypto-ed25519` just tests batches 0...15). I don't know of any reason larger batches would behave differently but we might want to extend this test to fully maintain test coverage.

Reply:
@randombit, so that I can merge, I'm resolving this open conversation because you approved the MR.